### PR TITLE
#162342447  Add test to comment expression model

### PR DESCRIPTION
**Pull request by ekpangmichael**
#### What does this PR do?
Implement Unit test for sequelize models
#### Description of Task to be completed?
- Write unit test to check for relationship and properties
- Install sequelize-test-helpers and  sinon package
#### How should this be manually tested?
Clone the repo,  and run `` npm install  `` ``  npm run test``
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
test sample

### CODE REVIEW

**Review by V-84 — COMMENTED** [review]
Hi Michael, nice work. However, replacing the checkModeName with an it()block makes it easier to customize the display messages and create multiple assertions if you want to.

File: tests/commentExpressions.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            expect(CommentExpressionModel.belongsTo.calledWith(Comment)).to.equal(true)
+        })
+    })
+
+    context('Comment Expression associations should', () => {
+        const User = user(sequelize, dataTypes);
+        before(() => {
+            CommentExpressionModel.associate({ User })
+        })
+        it('defined a belongsTo association with User model', () => {
```
Comment (V-84) [inline tests/commentExpressions.test.js]: Also, messages in the it() block should start with 'should'

File: tests/commentExpressions.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        it('should have property emotion', () => {
+            expect(instance).to.have.property('emotion');
+        })
+    })
+    context('Comment Expression associations should', () => {
+        const Comment = comment(sequelize, dataTypes);
+        before(() => {
+            CommentExpressionModel.associate({ Comment })
+        })
+        it('defined a belongsTo association with Comment model', () => {
```
Comment (V-84) [inline tests/commentExpressions.test.js]: messages in the it() block should start with 'should'

**Review by efalayi — APPROVED** [review]
LGTM

**Pull request merged**